ts/Grid/Core/Table/Actions/RowsVirtualizer.ts

```diff
@@ -469,21 +471,32 @@ class RowsVirtualizer {
      * once on initialization.
      */
     private getDefaultRowHeight(): number {
-        const mockRow = makeHTMLElement('tr', {
+        const vp = this.viewport;
+        const columns = vp.columns || {};
+        const mockHTMLRow = makeHTMLElement('tr', {
```

Comment:
Since we are creating a `mockRow`, wouldn't it be better to use it (just by adding additional css classes to it) instead of performing all this logic just to to copy the formatted value to different `mockHTMLRow` element?